hansol/app.js

```diff
@@ -19,23 +19,51 @@ const myDataSource = new DataSource({
   database: process.env.TYPEORM_DATABASE,
 });
 
-myDataSource
+dataSource
   .initialize()
 
   .then(() => {
-    console.log('Data Source has been initialized');
+    console.log("Data Source has been initialized");
   })
   .catch((err) => {
-    console.error('Error during Data Source initialization', err);
-    myDataSource.destroy();
+    console.error("Error during Data Source initialization", err);
+    dataSource.destroy();
   });
 
 app.use(express.json());
 app.use(cors());
-app.use(morgan('dev'));
+app.use(morgan("dev"));
 
-app.get('/ping', (req, res) => {
-  res.json({ message: 'pong' });
+app.get("/ping", (req, res) => {
+  res.json({ message: "pong" });
+});
+
+app.get("/users", async (req, res) => {
+  await dataSource.query(
+    `SELECT
+        users.name,
+        users.email,
+        users.password
+    FROM users`,
+    (err, rows) => {
+      res.status(200).json(rows);
+    }
+  );
+});
+
+app.post("/signup", async (req, res) => {
+  const { name, email, password } = req.body;
+
+  await dataSource.query(
+    `INSERT INTO users(
+		    name,
+		    email,
+		    password
+		) VALUES (?, ?, ?);
+		`,
+    [name, email, password]
+  );
```

Comment:
한솔님 ! 어제와 마찬가지로 띄어쓰기와 탭을 혼용해서 사용하게 되면, github상에서 indentation이 의도와 다르게 설정될 수 있습니다. 해당 부분 수정해서 push 해 주세요!
```suggestion
  await dataSource.query(
    `INSERT INTO users(
          name,
          email,
          password
        ) VALUES (?, ?, ?);
        `, [name, email, password]
  );
```